lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -18,6 +18,11 @@ table.ccx-schedule {
     th, td {
```

Comment:
I've spot checked this SASS against the style guide, and I don't see any violations. Let me know if I missed any @talbs